@market/hooks/useRelevantMarket.ts

```diff
@@ -24,14 +26,32 @@ function isOfferLike(value: MarketModule): value is FixedPriceLike {
   return value.type === MARKET_TYPES.FIXED_PRICE && value.side === FIXED_SIDE_TYPES.OFFER
 }
 
-export const useRelevantMarket = (markets: readonly MarketModule[] = []) => {
-  const auctions: AuctionLike[] = markets.filter(isAuctionLike).sort(getLatestBlockNumber) // @BJ / @Dain: question --> is the latest block number always desired
-  const asks: FixedPriceLike[] = markets.filter(isAskLike).sort(getLatestBlockNumber)
-  const offers: FixedPriceLike[] = markets.filter(isOfferLike).sort(getLatestBlockNumber) // <-- *all* active offers should be relevant
+// export const useRelevantMarket = (markets: readonly MarketModule[] = []) => {
+//   const auctions: AuctionLike[] = markets.filter(isAuctionLike).sort(getLatestBlockNumber) // @BJ / @Dain: question --> is the latest block number always desired
+//   const asks: FixedPriceLike[] = markets.filter(isAskLike).sort(getLatestBlockNumber)
+//   const offers: FixedPriceLike[] = markets.filter(isOfferLike).sort(getLatestBlockNumber) // <-- *all* active offers should be relevant
+
+//   const ask = asks[0]
+//   const auction = auctions[0]
+//   const offer = offers[0]
+
+//   return { offer, ask, auction, auctions }
+// }
+
+export const useRelevantMarket = (
+  markets: ReturnType<typeof useToken>['markets'] = []
+) => {
+  const auctions = markets.filter(
+    (item) => item?.properties?.__typename === 'V3ReserveAuction'
+  )
+  const asks = markets.filter((item) => item?.properties?.__typename === 'V3Ask')
+  const offers = markets.filter((item) => item?.properties?.__typename === 'V1Offer')
 
   const ask = asks[0]
   const auction = auctions[0]
   const offer = offers[0]
```

Comment:
We should return all offers and not just offers[0]... all offers are relevant (this is changed in the PR I pushed yesterday... we'd need to merge from main or fix it here)